### PR TITLE
Migrate breadcrumb to null safety.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: packages/devtools_app/lib/src/screens/inspector/inspector_breadcrumbs.dart [hunk 1]
```diff
@@ -190,14 +189,14 @@ class _InspectorBreadcrumbData {
...
-  final InspectorTreeNode node;
-  final IconData alternativeIcon;
-  final String alternativeText;
+  final InspectorTreeNode? node;
+  final IconData? alternativeIcon;
+  final String? alternativeText;
   final bool isSelected;
 
-  String get text => alternativeText ?? node?.diagnostic?.description;
+  String? get text => alternativeText ?? node?.diagnostic?.description;
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/screens/inspector/inspector_breadcrumbs.dart:197]: make this return a non-nullable String

**Review by kenzieschmoll — APPROVED** [review]
lgtm with one comment

File: packages/devtools_app/lib/src/screens/inspector/inspector_breadcrumbs.dart (hunk 1 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/screens/inspector/inspector_breadcrumbs.dart]: done

**Comment by polina-c** [comment]
tested

**Pull request merged**